lib/transport/selenium-webdriver/index.js

```diff
@@ -250,8 +250,8 @@ class Transport extends BaseTransport {
   ////////////////////////////////////////////////////////////////////
   // Output related
   ////////////////////////////////////////////////////////////////////
-  async showConnectInfo({startTime, port, host, start_process, sessionInfo}) {
-    this.showConnectSpinner(`Connected to ${colors.stack_trace(start_process ? this.serviceName : host)} on port ${colors.stack_trace(port)} ${colors.stack_trace('(' + (new Date() - startTime) + 'ms)')}.`);
+  async showConnectInfo({startTime, portStr, host, start_process, sessionInfo}) {
+    this.showConnectSpinner(`Connected to ${colors.stack_trace(start_process ? this.serviceName : host)} on ${portStr} ${colors.stack_trace('(' + (new Date() - startTime) + 'ms)')}.`);
```

Comment:
why is this change needed?

Reply:
Because I passed in portStr here, so renamed the variable to indicate the same. Just a cosmetic change.

https://github.com/nightwatchjs/nightwatch/pull/3364/files/3552669606a13d826861d22ab0a1b9aac26e4c58#diff-8f21ab197ad25d09890ab4026a9f46eae7b907700e7b4d750175a15f80c08e7cR242

Reply:
but it's not just renaming the variable, it's removing the "port" prefix and the color formatting.

Reply:
fixed.